app/models/cmr.rb

```diff
@@ -53,7 +53,11 @@ def self.get_raw_concept(concept_id, revision_id = nil)
   # If Cmr revision id isn't the latest, we pull the latest and ingest it.
 
   def self.update_collections(current_user)
-    records = Record.where state: [Record::STATE_OPEN.to_s, Record::STATE_IN_ARC_REVIEW.to_s, Record::STATE_READY_FOR_DAAC_REVIEW.to_s], recordable_type: "Collection"
+    if current_user.mdq_curator?
```

Comment:
This is related to record refreshing, correct?  
Can the MDQ curators access that functionality through the UI?

Reply:
no it is turned off for now.   only admin functionality has that turned on, but we can turn this on for them pretty easily when we want.   This is the conversation I had with Alicia where some of the features where only available to admin's, so we'll need to think about whether we want a "admin" for mdq_curators or not.